src/OrleansServiceBus/Providers/Streams/EventHub/IEventHubQueueCache.cs

```diff
@@ -47,5 +47,10 @@ public interface IEventHubQueueCache : IQueueFlowController, IDisposable
         /// Send purge signal to the cache, the cache will perform a time based purge on its cached messages
         /// </summary>
         void SignalPurge();
+
+        /// <summary>
+        /// Cache monitor for current cache
+        /// </summary>
+        ICacheMonitor CacheMonitor { get; }
```

Comment:
Who uses this property?  The monitoring seems like an implementation detail, why should any outside system know or care?